tiff_2_png/tiff_2_png_converter.py

```diff
@@ -7,7 +7,6 @@
 import imagesize
 import math
 import tifffile as tiff
```

Comment:
maybe rename `prepare-files.py` to `prepare_files.py` as all of the other file names use underscores